lib/audited/auditor.rb

```diff
@@ -297,10 +293,6 @@ def run_conditional_check(condition, matching: true)
         true
       end
 
-      def auditing_enabled=(val)
-        self.class.auditing_enabled = val
-      end
```

Comment:
I'm OK with removing this method, but please add a line in the CHANGELOG file indicating this change and that the class method should be used instead.

Reply:
done